applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -196,6 +206,48 @@ private void initialize(final String pluginVersion, final String sdkKey, final R
             creativeDebuggerEnabledToSet = null;
         }
 
+        if ( targetingYearOfBirthToSet != null )
+        {
+            sdk.getTargetingData().setYearOfBirth( targetingYearOfBirthToSet <= 0 ? null : targetingYearOfBirthToSet );
+            targetingYearOfBirthToSet = null;
+        }
+
+        if ( targetingGenderToSet != null )
+        {
+            setTargetingDataGender( targetingGenderToSet );
```

Comment:
nit: Overall looks good, but I like to keep everything consistent so I would use `sdk.getTargetingData(). setTargetingDataGender()` and use a getter for the gender like here:
https://github.com/AppLovin/Unity-Plugin/blob/c90fa6d893f648b9cc2fea33c623b1a77bd5520d/Android/Android-Unity-Plugin/src/main/java/com/applovin/mediation/unity/MaxUnityPlugin.java#L1285

Same for the `maximumAdContentRating`